### PR TITLE
[ADLS] add vnet and tests

### PR DESCRIPTION
**Pull request by lewu-msft**
## Description
Added virtual network CRUD operations for account management.
(https://github.com/Azure/azure-rest-api-specs/commit/b66d57be1bf2ee01797fffc34584d87b00d6629a)
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/DataLake.Store/Management.DataLake.Store/Microsoft.Azure.Management.DataLake.Store.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataLake.Store</PackageId>
 		<Description>Provides Data Lake Store account and filesystem management capabilities for Microsoft Azure.</Description>
-		<Version>2.4.0-preview</Version>
+		<Version>2.5.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/DataLake.Store/Management.DataLake.Store/Microsoft.Azure.Management.DataLake.Store.csproj]: @lewu-msft Looks like you are only making additive changes here, theoretically you could release `2.4.1-preview` version, however I leave it your discretion

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please create a generate.ps1 similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1) and regenerate the code. Please commit all the changes the script generates.
Also, run `msbuild build.proj /t:Build /p:Scope=SDKs\DataLake.Store\Management/DataLake.Store` and commit all artifacts generated

**Comment by dsgouda** [comment]
@lewu-msft Please fix failing tests

**Comment by dsgouda** [comment]
@lewu-msft gentle ping.

**Comment by dsgouda** [comment]
@lewu-msft tests are still failing

**Comment by lewu-msft** [comment]
working on it. tests passed locally but fail after pushing to repo

File: src/SDKs/DataLake.Store/Management.DataLake.Store/generate.ps1 [hunk 2]
```diff
@@ -0,0 +1 @@
+powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "datalake-store/resource-manager" -PowershellInvoker  -AutoRestVersion "latest"
```
Comment (shahabhijeet) [inline src/SDKs/DataLake.Store/Management.DataLake.Store/generate.ps1]: @lewu-msft please model the generate.ps1 according to what you see over here
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1#L1

File: src/SDKs/DataLake.Store/Management.DataLake.Store/Microsoft.Azure.Management.DataLake.Store.csproj [hunk 3]
```diff
@@ -6,7 +6,7 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.DataLake.Store</PackageId>
 		<Description>Provides Data Lake Store account and filesystem management capabilities for Microsoft Azure.</Description>
-		<Version>2.4.0-preview</Version>
+		<Version>2.4.1-preview</Version>
```
Comment (shahabhijeet) [inline src/SDKs/DataLake.Store/Management.DataLake.Store/Microsoft.Azure.Management.DataLake.Store.csproj]: @lewu-msft please update package release notes.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
If you are introducing breaking changes, you will need to bump up the major version of your package.
Left minor comments that needs to be addressed

File: src/SDKs/DataLake.Store/Management.DataLake.Store/generate.ps1 (hunk 2 above)
Comment (lewu-msft) [reply src/SDKs/DataLake.Store/Management.DataLake.Store/generate.ps1]: @shahabhijeet Thx! updated

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**